d3log/Makefile

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Since we're in this ecosystem, you probably want to use `cargo-make`

Reply:
I will look at this, but I assume its safe to defer?